### ISSUE TITLE
Revert #8300 or make it optional

### ISSUE THREAD

**Issue by theAkito** [thunderbird/thunderbird-android#8485]
### Checklist

- [X] I have used the search function to see if someone else has already submitted the same feature request.
- [X] I will describe the problem with as much detail as possible.
- [X] This issue only contains a request for one single feature, **not** multiple (related) features.

### App version

8.0

### Problem you are trying to solve

Whenever I open the app drawer, I can see the unread amount for all e-mail accounts, even the ones, I do not want to explcitly see. As most users of the app, I have various e-mail addresses registered, of which some I only want to visit during certain days of the week, like for example the work account, I only want to be bothered about on weekdays.

### Suggested solution

Remove https://github.com/thunderbird/thunderbird-android/issues/8300 or make it optional.

Please, remove or make it optional ASAP or tell me how to manually patch it out, without breaking the application. Seeing these numbers drives me insane.

**Comment by kewisch** [comment]
I think this would be reasonable to make optional, I can understand your use case. I won't be able to give you an ASAP option unless we go with the more relaxed definition of "soon" when we can get through some of the release feedback. You could certainly find the code that shows the number in the sources and install a debug build.

**Comment by theAkito** [comment]
> I think this would be reasonable to make optional, I can understand your use case. I won't be able to give you an ASAP option unless we go with the more relaxed definition of "soon" when we can get through some of the release feedback. You could certainly find the code that shows the number in the sources and install a debug build.

Thank you very much. I will try to revert the associated pull request, when I get the time. For now, I will have to live with it, as I do not want to downgrade.

**Comment by Domilopment** [comment]
Hello! First, I want to say that I appreciate the unread count on the accounts screen; it’s a helpful feature. However, I understand how some users might prefer it to be optional, and I wouldn’t mind this either.

I’m not sure if this is the best place for this, as it might be more of a feature request, but I’d like to suggest an additional approach to this issue. In the "Fetching mail" settings, there could perhaps be an option to set something like the "Folder poll frequency" but by specific days of the week. This would allow users who only check certain folders occasionally to reduce the frequency of mail fetching on other days.

**Comment by theAkito** [comment]
> I’m not sure if this is the best place for this, as it might be more of a feature request, but I’d like to suggest an additional approach to this issue. In the "Fetching mail" settings, there could perhaps be an option to set something like the "Folder poll frequency" but by specific days of the week. This would allow users who only check certain folders occasionally to reduce the frequency of mail fetching on other days.

Precisely. Had something like that on my mind too, but it takes a lot of effort to open issues with big issue templates...

**Comment by sebbu2** [comment]
The badge "plus de 99" is big and hide the 2 letters of the account in question, so I cna only guess which icon is which email. Can this be fixed ?

**Comment by kewisch** [comment]
This could be fixed in the French translation. In English it is simply `99+`. There are a few other locales where this could be adjusted. Would you be willing to head over to https://hosted.weblate.org/projects/tb-android/ and suggest a better translation for the string? The string name in question is `navigation_drawer_folder_item_badge_count_greater_than_99`

**Comment by sebbu2** [comment]
On [navigation_drawer](https://hosted.weblate.org/translate/tb-android/navigation-drawer/fr/?q=) I see ">99" while in k-9 mail i see "Plus de 99".

Does it uses a different one or has it been changed since the 8.0 release ?

**Comment by shamim-emon** [comment]
> For the French translation, the change has already been made, though we haven’t done a release since. We’re planning another release before the end of the year, which will include updated translations. A beta is coming out today.
> 
> For now, the best workaround might be to use two separate instances of Thunderbird/K-9 Mail, each configured for specific accounts (e.g., work vs personal). This could be Thunderbird vs Thunderbird Beta, or Thunderbird alongside K-9 Mail. Android also offers features like multi-user profiles or "Private Space" that could help separate accounts completely.
> 
> [@shamim-emon](https://github.com/shamim-emon), thank you so much for taking the time to create a pull request for this feature. I really appreciate your initiative and the effort to move toward a solution. That said, I think this is a feature we need to approach with more long-term planning.
> 
> Your pull request addresses hiding the unread count, but it applies to both folders and accounts. From past experience, we know that different users will likely want variations—some may want unread counts hidden for accounts but not folders, others might want it hidden for certain accounts only. This could quickly evolve into a complex feature with multiple settings.
> 
> Looking at the broader picture, I think this aligns with a potential set of features around digital wellbeing, such as schedules for showing or hiding certain accounts—similar to iOS Mail’s focus modes. Given workarounds are available, I think the next step is to develop a comprehensive plan for how work/personal separation could look across the app. This doesn’t necessarily have to be led by MZLA, but it does require someone to take a broader view of the concept. [@shamim-emon](https://github.com/shamim-emon) let me know how you feel about this and if you'd want to develop a product plan around this. If not that is of course totally fine, we'll consider this for our roadmap next year.
> 
> If you have questions about any of the issues you're working on before the pull request, please do feel free to find us on https://matrix.to/#/#tb-android-dev:mozilla.org or comment on the respective issue. Thank you again for the effort you've put into making life easier for Thunderbird users!

@kewisch Thank you for the elaboration. Thunderbird is a great application that is helping many users across the globe and I'm always honored at any given opportunity to contribute. I'm curious about the product plan around this, however, I'm relatively a new contributor and there are still a lot of ground to cover. Therefore, I will require some suggestion guideline around this.
Right now I see below options:
1. Brainstorm and discussion on individual focus points/ issues & we create issues on each of them.
2. I'm also good with given set of issues to work on them too.
Please let me know your suggestion on this.
Thanks in advance.
